### PR TITLE
Update readme.md and other fixes

### PR DESCRIPTION
**Pull request by luizcmarin**
"## Further reading
- [Martin Fowler's article](https://martinfowler.com/articles/injection.html)."

change localization to  Documentation section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -495,11 +493,6 @@ $config = ContainerConfig::create()
 $container = new Container($config);
 ```
 
-## Further reading
-
-- [Martin Fowler's article](https://martinfowler.com/articles/injection.html).
-
-
 # Benchmarks
```
Comment (samdark) [inline README.md:496]: That is for internals.

File: LICENSE.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
-Copyright © 2008 by Yii Software (https://www.yiiframework.com/)
+Copyright © 2008 by Yii Software (<https://www.yiiframework.com/>)
 All rights reserved.
 
 Redistribution and use in source and binary forms, with or without
 modification, are permitted provided that the following conditions
 are met:
 
- * Redistributions of source code must retain the above copyright
+* Redistributions of source code must retain the above copyright
```
Comment (vjik) [inline LICENSE.md:8]: Align it vertically, like here: https://raw.githubusercontent.com/yiisoft/package-template/master/LICENSE.md

File: README.md [hunk 3]
```diff
@@ -530,37 +528,40 @@ benchUndefinedNonexistent...............R5 I4 [μ Mo]/r: 0.946 0.942 (μs) [μSD
 (best [mean mode] worst) = 0.113 [4,483.856 4,486.051] 0.117 (μs) 
 ⅀T: 246,612.096μs μSD/r 43.563μs μRSD/r: 1.336%
 ```
+
 > **Warning!**
-> These summary statistics can be misleading. 
+> These summary statistics can be misleading.
 > You should always verify the individual subject statistics before drawing any conclusions.
-
+>
```
Comment (vjik) [inline README.md:535]:
```suggestion
```

Not need.

File: README.md [hunk 4]
```diff
@@ -634,40 +638,16 @@ Generated MD-file example
...
-### Static analysis
+## Documentation
 
-The code is statically analyzed with [Psalm](https://psalm.dev/). To run static analysis:
+- [Internals](docs/internals.md)
 
-```shell
-./vendor/bin/psalm
-```
+## Support
```
Comment (vjik) [inline README.md:648]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -680,3 +660,10 @@ Maintained by [Yii Software](https://www.yiiframework.com/).
 [![Telegram](https://img.shields.io/badge/telegram-join-1DA1F2?style=flat&logo=telegram)](https://t.me/yii3en)
 [![Facebook](https://img.shields.io/badge/facebook-join-1DA1F2?style=flat&logo=facebook&logoColor=ffffff)](https://www.facebook.com/groups/yiitalk)
 [![Slack](https://img.shields.io/badge/slack-join-1DA1F2?style=flat&logo=slack)](https://yiiframework.com/go/slack)
+
+## License
+
+The Yii DI is free software. It is released under the terms of the BSD License.
```
Comment (vjik) [inline README.md:666]:
```suggestion
The Yii Dependency Injection is free software. It is released under the terms of the BSD License.
```

File: docs/internals.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+```shell
+./vendor/bin/rector
+```
+
+## Dependencies
+
+Use [ComposerRequireChecker](https://github.com/maglnet/ComposerRequireChecker) to detect transitive
+[Composer](https://getcomposer.org/) dependencies.
+
+To run the checker, execute the following command:
```
Comment (vjik) [inline docs/internals.md:42]:
```suggestion
[Composer](https://getcomposer.org/) dependencies:
```

File: docs/internals.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+## Static analysis
+
+The code is statically analyzed with [Psalm](https://psalm.dev/). To run static analysis:
+
+```shell
+./vendor/bin/psalm
+```
+
+## Rector
```
Comment (vjik) [inline docs/internals.md:28]:
```suggestion
## Code style
```

**Review by vjik — CHANGES_REQUESTED** [review]

File: docs/internals.md
Comment (vjik) [inline docs/internals.md:1]: Why remove "Code style" and "Dependencies" sections?

Comment (luizcmarin) [reply docs/internals.md:1]: I couldn't resolve the conflicts I created, so I'm trying to go back in time. I saw that in the original readme.md they are not included. Then I removed it.

Comment (luizcmarin) [reply docs/internals.md:1]: resolved

File: README.md (hunk 1 above)
Comment (vjik) [reply README.md:496]: @luizcmarin Benchmarks for internals

**Pull request merged**